src/fs/file.rs

```diff
@@ -377,6 +377,40 @@ impl<'dir> File<'dir> {
                 .is_some_and(|p| all_mounts().contains_key(p))
     }
 
+    /// Whether the directory represents a Btrfs subvolume
+    #[cfg(target_os = "linux")]
+    pub fn is_btrfs_subvolume(&self) -> bool {
+        self.metadata.file_type().is_dir()
+            && self.metadata.ino() == 256
```

Comment:
Could you add a comment on where the number 256 comes from? i.e. the inode number of the root item of every btrfs subvolume.